src/app/registration/contact-form/contact-form.component.ts

```diff
@@ -112,7 +112,7 @@ export class ContactFormComponent implements OnInit {
         email: ['', [Validators.required, Validators.pattern(Constants.emailValidationRegex)]],
         emailCheck: ['', [Validators.required]],
         phone: ['', [Validators.required]],
-        enablePhone: ['', [Validators.required]]
+        enablePhone: [false, [Validators.required]]
```

Comment:
potential issue here - `[Validators - required]` enforces the field as mandatory. I trust this has been tested and works despite this fact but if this field links to the phone opt-in field then technically it is not mandatory.

Reply:
That makes sense to me! Following your advise I removed the required validators from "phone" and "enablePhone" fields. I have done another round of testing and everything is still working as desired.